dotcom-rendering/src/server/handler.editionsCrossword.ts

```diff
@@ -0,0 +1,10 @@
+import type { RequestHandler } from 'express';
+import { validateAsEditionsCrosswordType } from '../model/validate';
+
+export const handleEditionsCrossword: RequestHandler = ({ body }, res) => {
+	const editionsCrosswords = validateAsEditionsCrosswordType(body);
+
+	res.status(200).json({
+		data: editionsCrosswords,
+	});
```

Comment:
We may just want to `.sendStatus(200)` here for now? That will be enough to know that the validation step has succeeded.